lib/customer.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'restaurant'
+
+class Customer
+  attr_reader :name, :order, :restaurant
+
+  def initialize(restaurant = Restaurant.new)
+    @restaurant = restaurant
+    @order = Hash.new(0)
+  end
+
+  def menu
+    restaurant.view_menu
+  end
+
+  def place_order(dish, quantity = 1)
+    raise "Please select a dish from the menu." if !restaurant.menu_include?(dish)
+    order[dish] += quantity
+    return "Thank you for ordering. Your order so far is #{total_cost_per_dish}."
```

Comment:
Metrics/LineLength: Line is too long. [81/80]<br>Style/RedundantReturn: Redundant `return` detected.